src/App.js

```diff
@@ -1,172 +1,180 @@
+import Bell_pic from "./assets/GreenBell.png";
 import axios from "axios";
 
 const refreshReservation = new EventSource(
-  "http://localhost:5001/reservations/refresh"
+    "http://localhost:5001/reservations/refresh"
 );
 
 const App = () => {
-  //Hooks:
-  const [day, setDay] = useState("");
-  const [reservations, setReservation] = useState([]);
-  const [disable, setDisable] = React.useState(false);
+    //Hooks:
+    const [day, setDay] = useState("");
+    const [reservations, setReservation] = useState([]);
+    const [disable, setDisable] = React.useState(false);
 
-  useEffect(() => {
-    let time = new Date();
-    setDay(time.toLocaleDateString());
-    // --> getting data from database when the component renders:
-    axios
-      .get("http://localhost:5001/reservations/")
-      .then((Response) => {
-        setReservation(formatAvailabilities(setAvailabilities(Response.data)));
-      })
-      .catch((error) => {
-        console.log(error);
-      });
-  }, [day]);
+    useEffect(() => {
+        let time = new Date();
+        setDay(time.toLocaleDateString());
+        // --> getting data from database when the component renders:
+        axios
+            .get("http://localhost:5001/reservations/")
+            .then((Response) => {
+                setReservation(formatAvailabilities(setAvailabilities(Response.data)));
+            })
+            .catch((error) => {
+                console.log(error);
+            });
+    }, [day]);
 
-  // returns only the availabilities from all reservations.
-  function setAvailabilities(reservations) {
-    return reservations.filter(
-      (reservation) => reservation.day === day && reservation.roomNr === 1
-    );
-  }
+    // returns only the availabilities from all reservations.
+    function setAvailabilities(reservations) {
+        return reservations.filter(
+            (reservation) => reservation.day === day && reservation.roomNr === 1
+        );
+    }
 
-  function notify(email, roomNr) {
-    setDisable(true);
-    let mailTo = {
-      to: email,
-      roomNr,
-    };
-    console.log(mailTo);
-    axios
-      .post("http://localhost:5001/emails/send-email", mailTo)
-      .then((res) => {
-        console.log(res);
-        console.log("Message Sent");
-        setTimeout(() => setDisable(false), 5000);
-        alert("Message Sent");
-      })
-      .catch(() => {
-        console.log("Email could not be sent");
-        alert("Email could not be sent");
-      });
-  }
+    function notify(email) {
+        setDisable(true);
+        let mailTo = {
+            to: email,
+        };
+        console.log(mailTo);
+        axios
+            .post("http://localhost:5001/emails/send-email", mailTo)
+            .then((res) => {
+                console.log(res);
+                console.log("Message Sent");
+                setTimeout(() => setDisable(false), 6000);
+                alert("Message Sent");
```

Comment:
Remove alert? I would just change the notify text after click

Reply:
I will keep it to make the User comfortable